pootle/apps/pootle_project/models.py

```diff
@@ -45,6 +45,7 @@
 
 RESERVED_PROJECT_CODES = ('admin', 'translate', 'settings')
 PROJECT_CHECKERS = {
+    "none": checks.TranslationChecker,
```

Comment:
I wonder if it would be better to instead return `None` here and adjust `TranslationProject` to check if project checker is `None` and bail out early in that case. That way we would be running or instantiating any checker at all (if I am not wrong that is the purpose of this PR).